src/js/control-bar/play-toggle.js

```diff
@@ -58,6 +59,39 @@ class PlayToggle extends Button {
   }
 
   /**
+   * This gets called whenever the player seeks. We want to change the replay button
+   * back to a play/pause button on any seek because you cannot seek to the very end
+   * of a video
+   *
+   * @param {EventTarget~Event} [event]
+   *        The event that caused this function to run.
+   *
+   * @listens Player#seeked
+   */
+  handleSeeked(event) {
+    // check to see if the video has ended yet
+    if (!this.hasClass('vjs-ended')) {
+      return;
+    }
+
+    // if it has set us back to a play/pause button
+    // depending on the current state
+    this.removeClass('vjs-ended');
+
+    if (this.player_.paused()) {
```

Comment:
if we're ended, we should already be paused, so, I think we can probably simply this. Maybe we can just call the `handlePause` handler so we don't duplicate work?

Reply:
I was thinking about it, but it seemed a bit weird to be calling a function made for being called by events manually. If you don't see a problem with it. I can go ahead and do it though.

Reply:
probably ok? Can always pull it out into a separate function that both methods can call.